puffin_egui/Cargo.toml

```diff
@@ -22,14 +22,14 @@ include = [
 
 [dependencies]
 chrono = "0.4"
-egui = "0.18.1"
+egui = "0.19.0"
 natord = "1.0.9"
 once_cell = "1.7"
 puffin = { version = "0.13.3", path = "../puffin", features = ["packing"] }
 serde = { version = "1.0", features = ["derive"], optional = true }
 vec1 = "1.8"
 
 [dev-dependencies]
-eframe = "0.18.0"
+eframe = "0.19.0"
 egui-macroquad = "0.11.0"
```

Comment:
you also need to update `egui-macroquad`

Reply:
Ah, I totally missed that because I only tested the change with eframe and the wgpu backend. Looking at the repo of `egui_macroquad` it seems like we have to wait until [this](https://github.com/optozorax/egui-macroquad/pull/24) is merged.

Reply:
@sidit77 still planing to follow up with this one?

Reply:
@TimonPost As far as I can tell `egui-macroquad` is still not updated. I'm still waiting for the relevant pull request (see previous comment) to get merged.

Reply:
How about we disable the macroquad demo until `egui_macroquad` is updated?

Reply:
What would be the best way to do this? Rename the example file to something like `macroquad.rs.dis` and comment out the dev-dependency?

Reply:
Seems good to me. I'll be doing some puffin work in next weeks so when its ready can just revert that change. Do add a comment with a little note on why its commented.